packages/components/app/styles/components/page-header.scss

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// PAGEHEADER
+//
+
+.hds-page-header {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  container-type: inline-size;
+}
+
+.hds-page-header__body {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem 1rem;
```

Comment:
Niptick: some values are expressed as `rem`, others as `px`. Since the convention is to use `px` unless it's a font-size, can we convert all the `rem`s to `px`?